CONTRIBUTING.md

```diff
@@ -22,14 +22,18 @@ and we use [rustfmt](https://github.com/rust-lang/rustfmt) to make our code clea
     ```shell
     cargo clippy -- -D warnings
     ```
-3. Make sure that no new dependencies duplicates appear. Run the following check
+3. Make sure that no new dependencies duplicates appear. Run the following check   
    ```shell
    cargo deny check bans
    ```
 4. Make sure that dependencies do not have known vulnerabilities. If they do, update them.
    ```shell
    cargo deny check advisories
    ```
+   If you have problems with plugin, install it manually
```

Comment:
Please change it to "install cargo deny" and put before item 3.

Reply:
Now "Install cargo deny" goes before item 3.